notification/src/controller/index.js

```diff
@@ -0,0 +1,50 @@
+var Service = require('../services');
+var notificationService = new Service();
```

Comment:
Is there any specific reason to create a new instance everytime?  Can't this be a singleton?